ci/bandit.config

```diff
@@ -30,7 +30,6 @@
 # B306 : mktemp_q
 # B307 : eval
 # B308 : mark_safe
-# B309 : httpsconnection
```

Comment:
What's the reason for the changes in this file?

Reply:
The bandit fix (d64db3f414a9f6f2a89c0c2e4f9da8add6ddb1de) that was landed to master was needed in order to get CI tests to run, this has not yet been merged into the feature branch so I merged the commit into this PR.